Userland/Applications/PixelPaint/PixelPaintWindow.gml

```diff
@@ -43,59 +43,80 @@
             }
         }
 
-        @GUI::Widget {
-            fill_with_background_color: true
+        @GUI::DynamicWidgetContainer {
+            section_label: "Tools"
```

Comment:
Would it be better to rename this to 'Editor Panels', 'Studio Panels' or something?
Currently the box on the left with all of the brushes and selection tools is called the Tool Panel, so it might be a little confusing (in terms of documentation).